api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateNetworkOfferingCmd.java

```diff
@@ -122,6 +122,11 @@
             description = "true if network offering is meant to be used for VPC, false otherwise.")
     private Boolean forVpc;
 
+    @Parameter(name = ApiConstants.FOR_TUNGSTEN,
+            type = CommandType.BOOLEAN,
+            description = "true if network offering is meant to be used for TUNGSTEN, false otherwise.")
```

Comment:
Consider calling out "Tungsten Fabric" instead of just TUNGSTEN as this is end user facing and may not be obvious for users what Tungsten is otherwise.

Reply:
```suggestion
            description = "true if network offering is meant to be used for Tungsten-Fabric, false otherwise.")
```